pyproject.toml

```diff
@@ -2,6 +2,49 @@
 requires = [
     "setuptools>=42",
     "wheel",
-    "pybind11~=2.6",
+    "pybind11==2.12.0",
+    "tomli>=0.10; python_version<'3.11'",
 ]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "mapbox_earcut"
+version = "1.0.2"
+requires-python = ">=3.7"
```

Comment:
I think this should be >=3.9:

1) 3.7 is getting pretty old

2) numpy 2.0 isn't available for < 3.9 anyway.